easybuild/tools/github.py

```diff
@@ -383,17 +387,25 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
                 _log.info("Downloading %s from %s" % (fn, full_url))
                 download_file(fn, full_url, path=os.path.join(path, patched_file), forced=True)
         else:
-            apply_patch(diff_filepath, path, level=1)
+            apply_patch(diff_filepath, dl_path, level=1)
+
+    if dl_path != path:
+        for eb_file in glob.glob(os.path.join(dl_path, 'easybuild', 'easyconfigs', '*')):
```

Comment:
these are not files, but directories, and better use `os.listdir` since we'll symlink all of them anyway